llvm/include/llvm/CodeGen/LiveRegUnits.h

```diff
@@ -112,13 +112,13 @@ class LiveRegUnits {
   /// The regmask has the same format as the one in the RegMask machine operand.
   void addRegsInMask(const uint32_t *RegMask);
 
-  /// Returns true if no part of physical register \p Reg is live.
-  bool available(MCPhysReg Reg) const {
```

Comment:
I think available should stay as it is here (with rename)? A separate helper can combine the isReserved check

Reply:
The reason to keep this naming was to keep it analogous to the functions in LivePhysRegs (where "available" has the isReserved check and "contains" is also there) so that porting from LivePhysRegs to LiveRegUnits in other files can be smoother. 
Would you have suggestions on a better name for these two functions (available and contains)

Reply:
Just keep the available implementation in the header, it's trivia

Reply:
+1 on keeping `available`